README.md

```diff
@@ -43,34 +44,40 @@ When generating a package(s), the tooling will detect and generate all dependent
 
 After generating new reference packages, all new projects must be referenced as a
 [DependencyPackageProjects](https://github.com/dotnet/source-build-reference-packages/blob/main/eng/Build.props#L9).
-These must be defined in dependency order. There is a [tracking issue](https://github.com/dotnet/source-build/issues/1690)
-to address this manual step.
+These must be defined in dependency order. There is a
+[tracking issue](https://github.com/dotnet/source-build/issues/1690) to address this manual step.
 
-The tooling does not handle all situations and sometimes the generated code will need manual tweeks to get it to compile.
-If this occurs when generating a newer version of an existing package, it can be helpful to regenerate the older version
-to see what changes were made.
-
-The tooling has evolved over time and therefore when generating new packages, you may see edits made to existing packages.
-This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
-when it was originally generated.
+The tooling does not handle all situations and sometimes the generated code will need manual tweeks to get
+it to compile. If this occurs when generating a newer version of an existing package, it can be helpful to
+regenerate the older version to see what customizations to the generated code were made.
 
 #### Workflow
 
 * Generate reference package and its depencencies running the `./generate.sh --package <package>,<version>` script.
-* Revert changes for packages that were already existed in the repository.
-* Add `DependencyPackageProjects` for all new projects in the [eng/Build.props](https://github.com/dotnet/source-build-reference-packages/blob/main/eng/Build.props#L9)
+* Inspect any changes to packages that already existed in the repository. There are two reasons why previously
+generated packages show changes when being regenerated.
+    1. The package contains intentional code modifications on top of the generated code. This may be upgrading a
+    project reference to address a CVE or code fixups because the generate tooling does not support a scenario.
+    When this occurs, there should be code comments explaining why the code modification was made. If this is
+    the case, the changes to the existing package should be reverted.
+    2. The generate tooling has changed since the last time this package was generated. The new changes should
+    be considered better/correct and should be committed.
```

Comment:
This may end up being difficult for newbies to differentiate. We should strive to add comments to the source whenever making manual modifications so they are easy to spot.

Reply:
Agreed.  Hopefully the last step I added will help ensure comments are added.